main.py

```diff
@@ -71,6 +71,14 @@ def clear_labels(self):
             self.delete(text_id)
         self.labels = []
 
+    #STEPHEN MOSS ADDITION
```

Comment:
No need to add this comment since it's stored in the github meta.

Reply:
fair point, derp